download-paks

```diff
@@ -0,0 +1,550 @@
+#! /usr/bin/env bash
+
+# Retrieve Unvanquished resources from CDN
+# Arguments:
+# 1. Destination directory
+# 2. Cache file directory
+# For system-wide installation, you probably want something like
+# /usr/lib/games/unvanquished/pkg
+# /var/cache/games/unvanquished
+
+# Requirements: GNU coreutils, grep, sed, awk, diff,
+# and a download tool like aria2c, curl or wget.
+
+set -e
+
+# Work around a reported locale problem
+export LANG=C.UTF-8
+
+xdgHomeDir="${XDG_DATA_HOME:-${HOME}/.local/share}/unvanquished"
+
+# Version of Unvanquished for which this script is built
+version=0.51.1
```

Comment:
Can we not hard code this? Maybe fetch from latest tag or something?

Reply:
This is rewritten by `update-version-number` in any way.

Note that this script was shipped by debian package in the past, and it may be used outside of source dir in a legit way (at least downloaded by direct download).

What we can do is to read the git tag if exists, or fallback on this hardcoded one.